package.json

```diff
@@ -20,11 +20,22 @@
   "author": "Kurzdor",
   "license": "MIT",
   "dependencies": {
-    "ansi-red": "^0.1.1",
+    "concolor": "^0.1.11",
+    "eslint-config-prettier": "^4.1.0",
```

Comment:
eslint and eslint plugin must be devDeps because they are used only in development.